config/webpack/environment.js

```diff
@@ -1,5 +1,7 @@
 const { environment } = require('@rails/webpacker')
-const jquery = require('./plugins/jquery')
 
-environment.plugins.prepend('jquery', jquery)
+environment.loaders.get('sass').use.splice(-1, 0, {
```

Comment:
Moving away from webpacker so this will probably need to be modified for that once it's merged.